media/css/plugincheck/plugincheck-redesign.less

```diff
@@ -0,0 +1,602 @@
+                padding: 0 0 @baseLine;
+                width: auto;
+            }
+        }
+    }
+
+    .plugin-detail {
+        .open-sans-light;
+    }
+
+    small {
+        .font-size(@largeFontSize);
+        .open-sans-light;
+        display: block;
+        line-height: 1.5;
+    }
+
+    .plugin-meta span {
+        display: inline-block;
+
+        &:first-child {
+            margin-right: 1em;
+        }
+    }
+}
+
+.html-rtl #sec-plugin-featured {
+    .plugin-meta span:first-child {
+        margin-right: 0;
+        margin-left: 1em;
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Newsletter
+
+.newsletter-container {
+    margin-bottom: 30px;
+
+    #newsletter-subscribe {
+        .form-title {
+            padding-top: (@baseLine / 2);
+
+            @media only screen and (max-width: @breakDesktop) {
+                padding-top: 30px;
+            }
+
+            @media only screen and (max-width: @breakTablet) {
+                padding-top: 120px;
+            }
+
+            h3 {
+                .font-size(52px);
+                margin-bottom: 0;
+
+                @media only screen and (max-width: @breakDesktop) {
+                    .font-size(38px);
+                }
+
+                span {
+                    font-weight: bold;
+                }
+            }
+        }
+
+        h4 {
+            .font-size(32px);
+            color: #fff;
+            line-height: 1.2;
+            text-shadow: none;
+
+            @media only screen and (max-width: @breakDesktop) {
+                .font-size(24px);
+            }
+        }
+
+        // overrides styles from fxnewsletter-subscribe.less
+        #newsletter-form-thankyou {
+            float: none;
+            margin-left: auto;
+            margin: 0 auto;
+            text-align: center;
+            width: 50%;
+
+            @media only screen and (max-width: @breakTablet) {
+                width: 90%;
+            }
+
+            h3 {
+                .font-size(32px);
+                text-shadow: none;
+            }
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Plugin FAQ
+
+.plugin-faq-container {
+    margin-bottom: @baseLine * 2;
+
+    header {
+        .clearfix;
+        margin-bottom: @baseLine;
+    }
+
+    h2 {
+        float: left;
+        max-width: 45%;
+    }
+
+    .support-link {
+        .font-size(18px);
+        float: right;
+        max-width: 45%;
+    }
+
+    p,
+    ul li {
+        .font-size(@largeFontSize);
+        max-width: 40em;
+    }
+
+
+    @media only screen and (max-width: @breakTablet) {
+        h2,
+        .support-link {
+            float: none;
+            max-width: none;
+        }
+    }
+
+    span.menu:before {
+        content:'\2630 ';
+    }
+
+    span.menurev:after {
+        content:' \2630';
+    }
+}
+
+html[lang|="en"] {
+    .plugin-faq-container h2 {
```

Comment:
In some of the cases you called out here it is most certainly unnecessary still, so thanks for pointing out.